docs/api/v1/description_api.md

```diff
@@ -1,26 +1,29 @@
-# Paramètres généraux de l'API
-
 L'API de RDV-Solidarités vous permet de lire des données dans notre base depuis votre logiciel.
 
 Toutes les fonctionnalités de RDV-Solidarités ne sont pas encore disponibles via l’API. Contactez-nous si vous avez besoin de fonctionnalités qui ne sont pas encore présentes.
 
-## Authentification
+# Requêtes
```

Comment:
C'est bizarre d'avoir autant de `h1`. Pourquoi ne pas avoir gardé des `h2` ici ?

Reply:
Bloquant, je ne dirait pas ça, mais il y a un `H1` au-dessus il me semble, et il ne peut y avoir qu'un h1 par section ou par body, c'est donc du HTML non valide qui est produit ici. C'est un élément pour le RGAA également.

Reply:
Il y a peut être un truc qui m'échappe. D'après moi : 
- il n'y a pas de problème sémantique à avoir plusieurs sections h1 dans un body ou dans une page web, c'est permis depuis HTML5, comme [expliqué ici](https://webdesign.tutsplus.com/articles/the-truth-about-multiple-h1-tags-in-the-html5-era--webdesign-16824) (même si j'ai pas tout lu, c'est touffu)
- même si ça ne nous concerne pas trop, [ce n'est pas un problème pour le référencement](https://www.emaginance.com/plusieurs-balises-h1/)
- le title réel de la page est un h2, et on n'a pas vraiment la main dessus, c'est rendu par Swagger UI
- le HTML produit me semble tout à fait correct : 

![image](https://user-images.githubusercontent.com/1193334/202398311-ce47959a-46d4-4e9a-81cd-45561f50c55f.png)

En ce qui concerne le RGAA, les avis divergent !
- [ici](https://www.digitaweb.com/blog/rgaa-criteres-accessibilite-1) il est dit que le RGAA encourage l'usage d'un seul h1 par page
- [là](https://elbuffeto.github.io/100daysofa11y-day22/) au contraire, on indique qu'on peut utiliser plusieurs h1 dans la page

Si on reprend la [version 4.1 du RGAA](https://accessibilite.numerique.gouv.fr/doc/RGAA-v4.1.pdf), il n'y est pas fait mention du nombre de h1 : 

![image](https://user-images.githubusercontent.com/1193334/202399424-40d7505c-f791-40a7-b419-859079f7cd3d.png)

Après, j'ai totalement pu rater une info. J'ai également cherché dans [les critères](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/) mais je n'ai rien trouvé. Est-ce que tu as une info de ton côté ?

Reply:
Merci beaucoup pour toute la documentation.

J'avais en tête que l'on pouvait avoir plusieurs `<h1>` à condition de n'en avoir qu'un par `<section>`. Mais c'est effectivement sans doute une croyance obsolète.

Pour l'accessibilité, j'ai écrit un bout de test rapide pour vérifier, et il y a 5 violations découvertes avec l'outil automatique (peut-être plus si l'audit est fait par une personne humaine).

```
    1.1) Failure/Error: expect(page).to be_axe_clean                                                                                                                                                              
                                                                                                                                                                                                                   
            Found 5 accessibility violations:                                                                                                                                                                      
                                                                                                                                                                                                                   
            1) document-title: Documents must have <title> element to aid in navigation (serious)                                                                                                                  
                https://dequeuniversity.com/rules/axe/4.3/document-title?application=axeAPI                                                                                                                        
                The following 1 node violate this rule:                                                                                                                                                            
                                                                                                                                                                                                                   
                    Selector: html                                                                                                                                                                                 
                    HTML: <html>                                                                                                                                                                                   
                    Fix any of the following:                                                                                                                                                                      
                    - Document does not have a non-empty <title> element                                                                                                                                           
                                                                                                                                                                                                                   
            2) html-has-lang: <html> element must have a lang attribute (serious)                                                                                                                                  
                https://dequeuniversity.com/rules/axe/4.3/html-has-lang?application=axeAPI                                                                                                                         
                The following 1 node violate this rule:                                                                                                                                                            
                                                                                                                                                                                                                   
                    Selector: html                                                                                                                                                                                 
                    HTML: <html>                                                                                                                                                                                   
                    Fix any of the following:                                                                                                                                                                      
                    - The <html> element does not have a lang attribute                                                                                                                                            
                                                                                                                                                                                                                   
            3) landmark-one-main: Document should have one main landmark (moderate)                                                                                                                                
                https://dequeuniversity.com/rules/axe/4.3/landmark-one-main?application=axeAPI                                                                                                                     
                The following 1 node violate this rule:                                                                                                                                                            
                                                                                                                                                                                                                   
                    Selector: html                                                                                                                                                                                 
                    HTML: <html>                                                                                                                                                                                   
                    Fix all of the following:                                                                                                                                                                      
                    - Document does not have a main landmark                                                                                                                                                       
                                                                                                                                                                                                                   
            4) page-has-heading-one: Page should contain a level-one heading (moderate)                                                                                                                            
                https://dequeuniversity.com/rules/axe/4.3/page-has-heading-one?application=axeAPI                                                                                                                  
                The following 1 node violate this rule:                                                                                                                                                            
                                                                                                                                                                                                                   
                    Selector: html                                                                                                                                                                                 
                    HTML: <html>                                                                                                                                                                                   
                    Fix all of the following:                                                            
                    - Page must have a level-one heading                                                                                                                                                           
                                                                                                         
            5) region: All page content should be contained by landmarks (moderate)                      
                https://dequeuniversity.com/rules/axe/4.3/region?application=axeAPI                      
                The following 1 node violate this rule:                                                  
                                                                                                                                                                                                                   
                    Selector: pre                                                                        
                    HTML: <pre style="word-wrap: break-word; white-space: pre-wrap;">                    
                    Fix any of the following:       
                    - Some page content is not contained by landmarks               

```

Et il y a bien une violation sur les titres apparemment https://dequeuniversity.com/rules/axe/4.3/page-has-heading-one?application=axeAPI

Mais nous pouvons voir ça plus tard, en ajoutant le test automatisé. Il y aura peut-être des trucs sur lesquels nous n'allons pas pouvoir intervenir (nous avons un pépin d'accessibilité avec Chart.js par exemple)

Je ne souhaite pas bloquer la PR, c'était plus un commentaire, et un espoir que nous pourrons travailler sur ces points plus tard.

Reply:
D'ac, merci pour les tests et les précisions.